tests/dualtor/test_orch_stress.py

```diff
@@ -0,0 +1,322 @@
+
+
+def mux_state_configs(mux_intfs, state, op='SET'):
+    """Generate mux cable state configs
+
+    Args:
+        mux_intfs (list): List of interface names that are connected to mux cable.
+            Example: ['Ethernet0', 'Ethernet4', ...]
+        state (str): State of mux cable, allowed values: 'active', 'standby'
+        op (str, optional): Operation to be performed by swssconfig, allowed values: 'SET', 'DEL'. Defaults to 'SET'.
+
+    Returns:
+        list of dict: Returns mux state configs in a list of dict. Example:
+            [
+                {
+                    'MUX_CABLE_TABLE:Ethernet0':
+                        {
+                            'state': 'active'
+                        },
+                    'OP': 'SET'
+                },
+                {
+                    'MUX_CABLE_TABLE:Ethernet4':
+                        {
+                            'state': 'active'
+                        },
+                    'OP': 'SET'
+                },
+                ...
+            ]
+    """
+    configs = []
+    for intf in mux_intfs:
+        config = {
+            'MUX_CABLE_TABLE:{}'.format(intf):
+                {
+                    'state': state,
+                },
+            'OP': op
+        }
+        configs.append(config)
+    return configs
+
+
+def load_swss_config(dut, config_file):
+    """Load swss config file specified by 'config_file' in swss docker using command 'swssconfig'
+
+    Args:
+        dut (obj): Object for interacting with DUT.
+        config_file (str): Path and filename of the config file in swss docker container.
+    """
+    logger.info('Loading swss config {} ...'.format(config_file))
+    dut.shell('docker exec swss sh -c "swssconfig {}"'.format(config_file))
+    wait(2, 'for CRMs to be updated')
+    logger.info('Loading swss config {} done'.format(config_file))
+
+
+def _swss_path(filename):
+    """Helper function to transform swss config file path.
+
+    Copy files using 'docker cp swss' won't work if the destination path in 'swss' docker is '/tmp'.
+    This helper function is to transform path like '/tmp/filename.json' to '/filename.json'.
+
+    Args:
+        filename (str): Filename
+
+    Returns:
+        str: '/' + basename of filename.
+    """
+    return os.path.join('/', os.path.basename(filename))
+
+
+@pytest.fixture(scope='module')
+def swss_config_files(rand_selected_dut, tor_mux_intfs):
+    """This fixture is to generate/cleanup the swss config files in the swss docker.
+
+    Args:
+        rand_selected_dut (obj): Object for interacting with DUT.
+        tor_mux_intfs (list): List of interface names that are connected to mux cable.
+            Example: ['Ethernet0', 'Ethernet4', ...]
+    """
+    dut = rand_selected_dut
+    swss_configs = [
+        ('active', 'SET', SWSS_MUX_STATE_ACTIVE_CONFIG_FILE),
+        ('standby', 'SET', SWSS_MUX_STATE_STANDBY_CONFIG_FILE),
+        ('active', 'DEL', SWSS_MUX_STATE_DEL_CONFIG_FILE),
+    ]
```

Comment:
The state in config db should be ```standby``` after applying the second config. Will the last config (```'active', 'DEL'``` ) work? Should it be (```'standby', 'DEL'``` ) ?

Reply:
I just found that the ```active``` config is applied after ```standby```, and the ```DEL``` config is never used. Please ignore.